aiohttp/client_reqrep.py

```diff
@@ -781,39 +790,34 @@ def __init__(
         continue100: Optional["asyncio.Future[bool]"],
         timer: Optional[BaseTimerContext],
         request_info: RequestInfo,
-        traces: List["Trace"],
+        traces: Optional[List["Trace"]],
```

Comment:
Should keep the signature the same in 3.11 since it's technically breaking to change it. Wait for 3.12 for this part and split to a new pr

Reply:
revert change in https://github.com/aio-libs/aiohttp/pull/10030/commits/91ca7bffda1c16c6e890afd718821abf12eec513